core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -115,7 +115,7 @@ public boolean getRequiresConfirmation() {
      * @return permission required for user to access this management link, in addition to {@link Jenkins#ADMINISTER}
```

Comment:
```suggestion
     * @return permission required for user to access this management link, in addition to {@link Jenkins#MANAGE}
```

Reply:
This is not correct. The default permission is still `Jenkins.ADMINISTER`

Reply:
suggestion from @jglick is clearer and more correct.